### PR TITLE
Fix: coinjoin coordinator responses in PascalCase

### PR DESCRIPTION
**Pull request by szymonlesisz**
## Description

Related to https://github.com/trezor/trezor-suite/pull/9160

Coordinator and Middleware responses in PascalCase

TODO:
- [x] update middleware
- [x] deploy coordinator with PascalCase responses on staging (trezor.io)
- [x] test it on staging coordinator. tested with @hynek-jina and @marekrjpolak 
- [x] write patch for current coordinator with camelCase responses [2559804](https://github.com/trezor/trezor-suite/pull/9210/commits/2559804af3a609551b9bee5b00512a3111eb8f43)
- [x] test patch on wasabi coordinator and staging coordinator

### CODE REVIEW

**Comment by szymonlesisz** [comment]
@vdovhanych could you pls sign macos middleware build?

**Comment by vdovhanych** [comment]
Done

File: packages/coinjoin/src/types/client.ts [hunk 1]
```diff
@@ -1,14 +1,23 @@
 import { SessionPhase, WabiSabiProtocolErrorCode } from '../enums';
-import { AllowedRange, CoordinationFeeRate, Round } from './coordinator';
+// import { AllowedRange, CoordinationFeeRate, Round } from './coordinator';
```
Comment (marekrjpolak) [inline packages/coinjoin/src/types/client.ts]: Could we delete this?

Comment (szymonlesisz) [reply packages/coinjoin/src/types/client.ts]: sure, [ec438c6](https://github.com/trezor/trezor-suite/pull/9210/commits/ec438c6b225d2df32fe7269597c57ead127df038) and [15ecb3f](https://github.com/trezor/trezor-suite/pull/9210/commits/15ecb3f6a314016e56b4446f706ea36e78adbe17)

**Review by marekrjpolak — APPROVED** [review]
After thorough review, I can safely say that small letters were replaced by large letters. Also it works.

**Pull request merged**